crypto/rand/rand_lib.c

```diff
@@ -183,6 +183,7 @@ size_t rand_drbg_get_entropy(RAND_DRBG *drbg,
         }
 
     } else {
+#if defined(OPENSSL_RAND_SEED_NONE)
         if (prediction_resistance) {
             /*
              * We don't have any entropy sources that comply with the NIST
```

Comment:
It's ok for the moment, but should be replaced by something more generic in the long term.

Reply:
I was thinking of removing this section: based on the good enough to seed is good enough to reseed argument.  I think I've mentioned before that I really dislike the RAND_SEED_NONE option.

Reply:
Removed.

Reply:
The also covers the none source with an external supplier case.